ci/scripts/build.sh

```diff
@@ -163,7 +138,7 @@ if [ ! -d "geode/build/reports/combined" ]; then
 fi
 
 pushd geode/build/reports/combined
-gsutil -q -m cp -r * gs://${TEST_RESULTS_DESTINATION}
+  gsutil -q -m cp -r * gs://${TEST_RESULTS_DESTINATION}
```

Comment:
Isn't this part of archive results now?

Reply:
Actually, what is build.sh still used for?

Reply:
And to your earlier comment, yes.

The develop pipeline pattern is: `execute_build.sh`, with the archive uploaded in `archive_results.sh`.

The pull request pipeline remains untouched, with `build.sh` performing both duties.